go.mod

```diff
@@ -101,7 +101,7 @@ require (
 	github.com/gobwas/glob v0.2.3
 	github.com/google/go-cmp v0.6.0
 	github.com/google/uuid v1.4.0
-	github.com/grafana/regexp v0.0.0-20221122212121-6b5c0a4cb7fd
+	github.com/grafana/regexp v0.0.0-20221122212121-6b5c0a4cb7fd // indirect
```

Comment:
nit: should this be moved to indirect group?

Reply:
idk go mod tidy did this.  It would surprise me if we were using grafana's regex library directly